www/VuforiaPlugin.js

```diff
@@ -1,5 +1,10 @@
 var VuforiaPlugin = {
-  startVuforia: function(imageFile ,imageTargets, overlayCopy, vuforiaLicense, imageFoundCallback, errorCallback){
+  startVuforia: function(options, imageFoundCallback, errorCallback){
+    var imageFile = options.databaseXmlFile;
+    var imageTargets = options.targetList;
+    var overlayCopy = options.overlayMessage;
+    var vuforiaLicense = options.vuforiaLicense;
+    var showAndroidCloseButton = options.showAndroidCloseButton?true:false;
```

Comment:
Yes of course!


2016-06-15 15:45:06 +0000 Matt Rayner <notifications@github.com> :

> In www/VuforiaPlugin.js
> <https://github.com/mattrayner/cordova-plugin-vuforia/pull/48#discussion_r67187900>
> :
>
> > @@ -1,5 +1,10 @@
> >  var VuforiaPlugin = {
> > -  startVuforia: function(imageFile ,imageTargets, overlayCopy, vuforiaLicense, imageFoundCallback, errorCallback){
> > +  startVuforia: function(options, imageFoundCallback, errorCallback){
> > +    var imageFile = options.databaseXmlFile;
> > +    var imageTargets = options.targetList;
> > +    var overlayCopy = options.overlayMessage;
> > +    var vuforiaLicense = options.vuforiaLicense;
> > +    var showAndroidCloseButton = options.showAndroidCloseButton?true:false;
>
> Can we change this to:
>
> var showAndroidCloseButton = options.showAndroidCloseButton ? true : false;
>
> This is a bit more readable
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mattrayner/cordova-plugin-vuforia/pull/48/files/959ffe424efed54d0ad8a77d32732397599147f0#r67187900>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ANWwnyuMYWtc1a6ha7jAzzSRh8LSEeHkks5qMB4CgaJpZM4I0lMX>
> .
>